wp-content/plugins/aesirx-analytics/aesirx-analytics.php

```diff
@@ -8,7 +8,10 @@
  * Author URI: https://aesirx.io/
  * Domain Path: /languages
  * Text Domain: aesirx-analytics
- * Requires PHP: 7.2
+ * Requires PHP: 8.1
```

Comment:
so we go from 7.2 to 8.1. is this really needed. I don't have exact number but are limit us with this to a smaller audience?

Reply:
i have checked and i see this is related to my local composer...
when i was testing the create of the release it created a requirement of the 8.1 PHP that is why i updated it ...
i have checked the release from @NguyenBao10 3.1.0 and the composer requires 7.4...
so maybe we need to change to 7.4...
![image](https://github.com/aesirxio/wordpress-analytics-plugin/assets/21126820/af799bf3-caa1-4902-a7d5-ae2d4062bb13)